src/data/membersData.js

```diff
@@ -28,8 +28,8 @@ const membersData = [
   },
   {
     id: 'd2c4875f-e721-4cba-be90-e0dc1538181a',
-    name: 'Hadi 2',
-    email: 'hadikhairallah@outlook.com',
+    name: 'Denis ',
+    email: 'denis.gillet@epfl.ch',
```

Comment:
Actually could you avoid using real data? Like anna@email.com. Names are okay, but mail a bit less.